### ISSUE TITLE
FM64 features implementation

### ISSUE THREAD

**Issue by dukesrg** [dukesrg/logue-osc#2]
- [x] 32 6-op algorithms
- [x] 8 4-op algorithms
- [x] Frequency modulation (modulation index issues)
- [x] Feedback
- [x] EG (rough rates)
- [x] EG shift
- [ ] Waveform
- [x] 6-op frequency/ratio coarse
- [x] 6-op frequency/ratio fine
- [x] 4-op frequency/ratio coarse
- [ ] 4-op frequency/ratio fine
- [x] Phase init
- [x] Transpose
- [ ] Detune
- [ ] PEG
- [ ] LFO
- [ ] Amplitude modulation
- [ ] Pitch modulation
- [ ] Rate scaling
- [ ] Portamento
- [ ] Reverb

**Comment by aframires** [comment]
Hey! If you need some help with this let me know!

**Comment by dukesrg** [comment]
@aframires Yes, thanks, we're actually stuck with EG level/Out level relation now.
Several features just won't fit into performance limit, but correct EG is a must.

Here is the current discussion in progress https://www.gearslutz.com/board/electronic-music-instruments-and-electronic-music-production/1306032-korg-logue-user-oscillator-programming-thread-22.html
